### ISSUE TITLE
Can't build OneDrive on Debian Stretch "Makefile:29: recipe for target 'onedrive' failed"

### ISSUE THREAD

**Issue by Kurgol** [skilion/onedrive#393]
I think this is a duplicate of #227, following is a log of me following the installation instructions on Debian Stretch:

```
$ sudo apt install libcurl4-openssl-dev
[sudo] password for kurgol: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libcurl4-openssl-dev is already the newest version (7.52.1-5+deb9u6).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
kurgol@hal:~$ sudo apt install libsqlite3-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libsqlite3-dev is already the newest version (3.16.2-5+deb9u1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
kurgol@hal:~$ sudo wget http://master.dl.sourceforge.net/project/d-apt/files/d-apt.list -O /etc/apt/sources.list.d/d-apt.list
--2018-07-24 09:15:37--  http://master.dl.sourceforge.net/project/d-apt/files/d-apt.list
Resolving master.dl.sourceforge.net (master.dl.sourceforge.net)... 216.105.38.12
Connecting to master.dl.sourceforge.net (master.dl.sourceforge.net)|216.105.38.12|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 116 [application/octet-stream]
Saving to: ‘/etc/apt/sources.list.d/d-apt.list’

/etc/apt/sources.li 100%[===================>]     116  --.-KB/s    in 0s      

2018-07-24 09:15:37 (4.77 MB/s) - ‘/etc/apt/sources.list.d/d-apt.list’ saved [116/116]

kurgol@hal:~$ sudo apt-get update && sudo apt-get -y --allow-unauthenticated install --reinstall d-apt-keyring
Ign:1 http://ftp.uk.debian.org/debian stretch InRelease
Ign:2 http://dl.google.com/linux/chrome/deb stable InRelease                   
Hit:3 http://netcologne.dl.sourceforge.net/project/d-apt d-apt InRelease       
Hit:4 http://ftp.uk.debian.org/debian stretch-updates InRelease                
Hit:5 http://dl.google.com/linux/chrome/deb stable Release                     
Hit:6 http://security.debian.org/debian-security stretch/updates InRelease     
Hit:7 http://ftp.uk.debian.org/debian stretch Release                          
Hit:8 https://packages.microsoft.com/repos/vscode stable InRelease             
Hit:9 http://packages.crunchbangplusplus.org/stretch stretch InRelease
Reading package lists... Done                      
Reading package lists... Done
Building dependency tree       
Reading state information... Done
0 upgraded, 0 newly installed, 1 reinstalled, 0 to remove and 0 not upgraded.
Need to get 3,354 B of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 http://netcologne.dl.sourceforge.net/project/d-apt d-apt/main amd64 d-apt-keyring all 2012.02.19-3.1 [3,354 B]
Fetched 3,354 B in 0s (47.4 kB/s)        
(Reading database ... 152986 files and directories currently installed.)
Preparing to unpack .../d-apt-keyring_2012.02.19-3.1_all.deb ...
Warning: apt-key should not be used in scripts (called from prerm maintainerscript of the package d-apt-keyring)
Unpacking d-apt-keyring (2012.02.19-3.1) over (2012.02.19-3.1) ...
Setting up d-apt-keyring (2012.02.19-3.1) ...
Warning: apt-key should not be used in scripts (called from postinst maintainerscript of the package d-apt-keyring)
OK
kurgol@hal:~$ sudo apt-get update && sudo apt-get install dmd-compiler dub
Ign:1 http://ftp.uk.debian.org/debian stretch InRelease
Ign:2 http://dl.google.com/linux/chrome/deb stable InRelease                   
Hit:3 http://netcologne.dl.sourceforge.net/project/d-apt d-apt InRelease       
Hit:4 http://ftp.uk.debian.org/debian stretch-updates InRelease                
Hit:5 http://dl.google.com/linux/chrome/deb stable Release                     
Hit:6 http://ftp.uk.debian.org/debian stretch Release                          
Hit:7 http://security.debian.org/debian-security stretch/updates InRelease     
Hit:8 https://packages.microsoft.com/repos/vscode stable InRelease             
Hit:10 http://packages.crunchbangplusplus.org/stretch stretch InRelease        
Reading package lists... Done                      
Reading package lists... Done
Building dependency tree       
Reading state information... Done
dmd-compiler is already the newest version (2.081.1-0).
dub is already the newest version (1.10.0-0).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
kurgol@hal:~$ git clone https://github.com/skilion/onedrive.git
Cloning into 'onedrive'...
remote: Counting objects: 961, done.
remote: Total 961 (delta 0), reused 0 (delta 0), pack-reused 961
Receiving objects: 100% (961/961), 283.30 KiB | 0 bytes/s, done.
Resolving deltas: 100% (667/667), done.
kurgol@hal:~$ cd onedrive/
kurgol@hal:~/onedrive$ make
echo v1.1.1-2-gc231b13 >version
dmd -g -ofonedrive -O -L-lcurl -L-lsqlite3 -L-ldl -J. src/config.d src/itemdb.d src/log.d src/main.d src/monitor.d src/onedrive.d src/qxor.d src/selective.d src/sqlite.d src/sync.d src/upload.d src/util.d
src/sqlite.d(71): Error: module `stdlib` is in file 'std/c/stdlib.d' which cannot be read
import path[0] = /usr/include/dmd/phobos
import path[1] = /usr/include/dmd/druntime/import
Makefile:29: recipe for target 'onedrive' failed
make: *** [onedrive] Error 1
```

**Comment by abraunegg** [comment]
Agree on that one - or even at least post a note advising what the current status is

**Comment by skilion** [comment]
Fixed in 19793c854d9dd97eb5accf94d21406f825c75fb5
